koku/reporting/migrations/0038_auto_20190220_1511.py

```diff
@@ -0,0 +1,290 @@
+                    ON tm.ocp_id = ocp.id
+            );
+            """
+        ),
+        migrations.RunSQL(
+            """
+            DROP VIEW IF EXISTS reporting_ocpawsstoragelineitem_daily;
+
+            CREATE OR REPLACE VIEW reporting_ocpawsstoragelineitem_daily AS (
+                WITH cte_storage_tag_matchted as (
+                    SELECT aws.id as aws_id,
+                            COALESCE(pvl.id, pvcl.id) as ocp_id,
+                            aws.usage_start,
+                            COALESCE(pvl.namespace, pvcl.namespace) as namespace
+                        FROM (
+                        SELECT aws.id,
+                            aws.usage_start,
+                            LOWER(key) as key,
+                            LOWER(value) as value
+                            FROM reporting_awscostentrylineitem_daily as aws,
+                                jsonb_each_text(aws.tags) labels
+                        ) AS aws
+                        LEFT JOIN (
+                            SELECT ocp.id,
+                                ocp.usage_start,
+                                ocp.cluster_alias,
+                                ocp.node,
+                                ocp.namespace,
+                                LOWER(key) as key,
+                                LOWER(value) as value
+                            FROM reporting_ocpstoragelineitem_daily as ocp,
+                                jsonb_each_text(ocp.persistentvolume_labels) labels
+                        ) AS pvl
+                            ON aws.usage_start::date = pvl.usage_start::date
+                                AND (
+                                    (aws.key = pvl.key AND aws.value = pvl.value)
+                                    OR (aws.key = 'ocp_cluster' AND aws.value = pvl.cluster_alias)
+                                    OR (aws.key = 'ocp_node' AND aws.value = pvl.node)
+                                    OR (aws.key = 'ocp_project' AND aws.value = pvl.namespace)
+                                )
+                        LEFT JOIN (
+                            SELECT ocp.id,
+                                ocp.usage_start,
+                                ocp.cluster_alias,
+                                ocp.node,
+                                ocp.namespace,
+                                LOWER(key) as key,
+                                LOWER(value) as value
+                            FROM reporting_ocpstoragelineitem_daily as ocp,
+                                jsonb_each_text(ocp.persistentvolumeclaim_labels) labels
+                    ) AS pvcl
+                            ON aws.usage_start::date = pvcl.usage_start::date
+                                AND (
+                                    (aws.key = pvcl.key AND aws.value = pvcl.value)
+                                    OR (aws.key = 'ocp_cluster' AND aws.value = pvcl.cluster_alias)
```

Comment:
Sorry about this, but per Dayle's comments we should use `openshift` not `ocp` so let's update these to:
- `openshift_project`
- `openshift_node`
- `openshift_cluster`